x/wasm/types/validation.go

```diff
@@ -5,12 +5,13 @@ import (
 )
 
 const (
-	MaxWasmSize = 500 * 1024
-
 	// MaxLabelSize is the longest label that can be used when Instantiating a contract
 	MaxLabelSize = 128
 )
 
+// MaxWasmSize is the largest a compiled contract code can be when storing code on chain
+var MaxWasmSize int = 800 * 1024
```

Comment:
Just curious why you increased the value to 800kb? Do you have a contract example with this size? We had some internal discussion for [tgrade](https://github.com/confio/tgrade/issues/239#issuecomment-1109603114) yesterday where I looked at some of our contracts and max was 555kb. 

```suggestion
var MaxWasmSize int = 777 * 1024
```

Reply:
ah, Ethan suggested 800kb. We should have coordinated better. 🤷  It would help with tgrade if you can make it 777kb

Reply:
777kb is an odd number. 
But tgrade can easily override this default, so maybe best to use 800 and verify we can change it in the imported one